code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -600,3 +600,92 @@
+
+/datum/chemical_reaction/mutationtoxin/lizard
+	name = /datum/reagent/mutationtoxin/lizard
+	id = /datum/reagent/mutationtoxin/lizard
+	results = list(/datum/reagent/mutationtoxin/lizard = 1)
+	required_reagents  = list(/datum/reagent/mutationtoxin/unstable = 1, /datum/reagent/liquidgibs = 1)
+
+/datum/chemical_reaction/mutationtoxin/felinid
+	name = /datum/reagent/mutationtoxin/felinid
+	id = /datum/reagent/mutationtoxin/felinid
+	results = list(/datum/reagent/mutationtoxin/felinid = 1)
+	required_reagents  = list(/datum/reagent/mutationtoxin/unstable = 1, /datum/reagent/toxin/fentanyl = 1, /datum/reagent/impedrezene = 1)
+	
+/datum/chemical_reaction/mutationtoxin/fly
+	name = /datum/reagent/mutationtoxin/fly
+	id = /datum/reagent/mutationtoxin/fly
+	results = list(/datum/reagent/mutationtoxin/fly = 1)
+	required_reagents  = list(/datum/reagent/mutationtoxin/unstable = 1, /datum/reagent/toxin/mutagen = 1)
+
+/datum/chemical_reaction/mutationtoxin/moth
+	name = /datum/reagent/mutationtoxin/moth
+	id = /datum/reagent/mutationtoxin/moth
+	results = list(/datum/reagent/mutationtoxin/moth = 1)
+	required_reagents  = list(/datum/reagent/mutationtoxin/unstable = 1, /datum/reagent/toxin/lipolicide = 1) //I know it's the opposite of what moths like, but I am out of ideas for this.
+	
+/datum/chemical_reaction/mutationtoxin/pod
+	name = /datum/reagent/mutationtoxin/pod
+	id = /datum/reagent/mutationtoxin/pod
+	results = list(/datum/reagent/mutationtoxin/pod = 1)
+	required_reagents  = list(/datum/reagent/mutationtoxin/unstable = 1, /datum/reagent/plantnutriment/eznutriment = 1)
+
+/datum/chemical_reaction/mutationtoxin/golem
+	name = /datum/reagent/mutationtoxin/golem
+	id = /datum/reagent/mutationtoxin/golem
+	results = list(/datum/reagent/mutationtoxin/golem = 1)
+	required_reagents  = list(/datum/reagent/mutationtoxin/unstable = 1, /datum/reagent/silver = 1)
+
+/datum/chemical_reaction/mutationtoxin/abductor
+	name = /datum/reagent/mutationtoxin/abductor
+	id = /datum/reagent/mutationtoxin/abductor
+	results = list(/datum/reagent/mutationtoxin/abductor = 1)
+	required_reagents  = list(/datum/reagent/mutationtoxin/unstable = 1, /datum/reagent/medicine/morphine = 1, /datum/reagent/toxin/mutetoxin = 1)
+	
+/datum/chemical_reaction/mutationtoxin/ipc
+	name = /datum/reagent/mutationtoxin/ipc
+	id = /datum/reagent/mutationtoxin/ipc
+	results = list(/datum/reagent/mutationtoxin/ipc = 1)
+	required_reagents  = list(/datum/reagent/mutationtoxin/unstable = 1, /datum/reagent/teslium = 1)
+	
+//////////////Mutatuion toxins made out of advanced toxin/////////////
+
+/datum/chemical_reaction/mutationtoxin/skeleton
+	name = /datum/reagent/mutationtoxin/skeleton
+	id = /datum/reagent/mutationtoxin/skeleton
+	results = list(/datum/reagent/mutationtoxin/skeleton = 1)
+	required_reagents  = list(/datum/reagent/aslimetoxin = 1, /datum/reagent/consumable/milk = 1, /datum/chemical_reaction/facid = 1) //Because acid melts flesh off.
```

Comment:
did you mean to write `/datum/reagent/toxin/acid/fluacid` (fluorosulfuric acid) instead of `/datum/chemical_reaction/facid = 1`?

Reply:
Oh, right. Just was really sleepy when coding that. I'll fix it later today when I get home.